### PR TITLE
Viki/import chargebee

### PR DESCRIPTION
**Pull request by VikiAnn**
This includes everything except the encrypted credentials file, it should be safe to push up but I wanted to double check that you're OK with that instead of dotenv. I figure going with Rails defaults as much as possible is probably a good idea.

Sorry the diff is so huge, that's mostly from moving all the gem stuff into its own directory.

### CODE REVIEW

**Comment by lilith** [comment]
Let's delete gem/lambda - it's got no future and is just a thousand vendored npm modules.

File: web/app/mailers/license_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+class LicenseMailer < ApplicationMailer
+  OUR_EMAILS = [
+    'nathanael.jones@gmail.com',
```
Comment (lilith) [inline web/app/mailers/license_mailer.rb]: lilith@imazen.io should be the new CC address.

**Comment by VikiAnn** [comment]
Would you like me to make a separate PR with the new gem directory and removed lambda directory? Might make this one easier to go through.

File: web/app/mailers/license_mailer.rb (hunk 1 above)
Comment (VikiAnn) [reply web/app/mailers/license_mailer.rb]: Should that replace all the other emails in there, or just that one?

**Comment by lilith** [comment]
Re: credentials, heroku and CI collectively need them to be stored in env vars, so any credential solution needs to be able to fallback to that if the dev environment file is missing.

**Comment by lilith** [comment]
> Would you like me to make a separate PR with the new gem directory and removed lambda directory? Might make this one easier to go through.

Sure, that would help.

**Comment by lilith** [comment]
> Re: credentials, heroku and CI collectively need them to be stored in env vars, so any credential solution needs to be able to fallback to that if the dev environment file is missing.

I read https://guides.rubyonrails.org/security.html#custom-credentials

What I dislike about that solution is that it requires local rails to update credentials (AFAICT), which could be a problem if production credentials expire and need to be updated remotely.  It also makes it harder to see a list of active credentials.

File: web/app/mailers/license_mailer.rb (hunk 1 above)
Comment (lilith) [reply web/app/mailers/license_mailer.rb]: Just that one. Although michael might want off the list too hehe.

**Comment by VikiAnn** [comment]
>What I dislike about that solution is that it requires local rails to update credentials (AFAICT), which could be a problem if production credentials expire and need to be updated remotely. It also makes it harder to see a list of active credentials.

Ahh gotcha, that makes sense. I'll switch that over then

**Comment by VikiAnn** [comment]
@lilith should be ready for another look now, it's passing in Travis so I think next step would be to merge and deploy it to Heroku and make sure things are working as expected

**Comment by lilith** [comment]
Looks great to me!

**Pull request merged**